gwf/colours.py

```diff
@@ -0,0 +1,10 @@
+COLORS = {
+    'red': '\033[91m',
+    'blue': '\033[94m',
+    'green': '\033[32m',
+    'black': '\033[30m',
+    'yellow': '\033[33m',
+    'bold': '\033[1m',
+}
+
+CLEAR = '\033[0m'
```

Comment:
I think the colours would be nicer as separate variables `RED = '\033[91m'` and so on since the rest of the code does nothing but constant lookups.
Maybe the usage code would be even better if they were functions:

    def red(s):
        return RED + s + CLEAR

If you chose not to do one of these changes the mismatch between the british spelling on the module name and the american spelling on the variable should probably be fixed :)